runtime/compiler/env/VMJ9.cpp

```diff
@@ -4726,6 +4726,9 @@ TR_J9VMBase::needsInvokeExactJ2IThunk(TR::Node *callNode, TR::Compilation *comp)
       && (  method->getMandatoryRecognizedMethod() == TR::java_lang_invoke_MethodHandle_invokeExact
          || method->isArchetypeSpecimen()))
       {
+      if (comp->isOutOfProcessCompilation())
```

Comment:
If you have overridden this method, while do you still need to modify the base class?

Reply:
No, this is a typo, forgot to delete it.